src/Illuminate/Foundation/Exceptions/Handler.php

```diff
@@ -692,6 +692,8 @@ protected function convertExceptionToArray(Throwable $e)
      * @param  \Symfony\Component\Console\Output\OutputInterface  $output
      * @param  \Throwable  $e
      * @return void
+     *
+     * @internal This method is not meant to be used or overwritten outside the framework.
```

Comment:
This PR marks a method internal which is part of an interface?

`class Handler implements ExceptionHandlerContract` -> https://github.com/laravel/framework/blob/05b785d7d24375b5437ea115a363f6a63364f84a/src/Illuminate/Contracts/Debug/ExceptionHandler.php#L38-L45